test/viewmodels/map/map_view_model_unit_test.dart

```diff
@@ -23,15 +26,51 @@ void main() {
   });
 
   test("On map create", () async {
-    //create a new instance of the map view model
-    final mapViewModel = MapViewModel();
-
-    final mockGoogleMapController = MockGoogleMapController();
-
     //create a new instance of the map controller
     mapViewModel.onMapCreated(mockGoogleMapController);
 
     //check if the map controller is created
     expect(mapViewModel.mapController.isCompleted, true);
   });
+
+  group("Camera", () {
+    test("Camera follows user by default", () async {
+      //create a new instance of the map controller
+      mapViewModel.onMapCreated(mockGoogleMapController);
+      when(mockGoogleMapController.animateCamera(any)).thenAnswer((_) async {});
+
+      //new position for the camera
+      const newLocation = LatLng(1, 1);
```

Comment:
I believe we can also put this variable in the setup.
Like that we don't need to recreate it in each test.

Reply:
Yes good idea!
I think it should also go in the mock data folder.

Reply:
I actually moved the initialization at the beginning of the main. Because it is a constant, we don't need to put it in setUp.
Refactored in 4a0191fee1bf5fafe3f1e8770a5ec38bfd6d3071